sqlglot/dialects/athena.py

```diff
@@ -79,27 +104,40 @@ class Parser(Trino.Parser):
             TokenType.USING: lambda self: self._parse_as_command(self._prev),
         }
 
+    class _HiveGenerator(Hive.Generator):
+        def preprocess(self, expression: exp.Expression) -> exp.Expression:
+            # package any ALTER TABLE ADD actions into a Schema object
+            # so it gets generated as `ALTER TABLE .. ADD COLUMNS(...)`
+            # instead of `ALTER TABLE ... ADD COLUMN` which is invalid syntax on Athena
+            if isinstance(expression, exp.Alter) and expression.kind == "TABLE":
+                if expression.actions and isinstance(expression.actions[0], exp.ColumnDef):
+                    new_actions = exp.Schema(expressions=expression.actions)
+                    expression.set("actions", [new_actions])
+
+            return super().preprocess(expression)
+
     class Generator(Trino.Generator):
         """
         Generate queries for the Athena Trino execution engine
         """
 
-        TYPE_MAPPING = {
-            **Trino.Generator.TYPE_MAPPING,
-            exp.DataType.Type.TEXT: "STRING",
+        PROPERTIES_LOCATION = {
+            **Trino.Generator.PROPERTIES_LOCATION,
+            exp.LocationProperty: exp.Properties.Location.POST_WITH,
         }
 
         TRANSFORMS = {
             **Trino.Generator.TRANSFORMS,
-            exp.FileFormatProperty: lambda self, e: f"'FORMAT'={self.sql(e, 'this')}",
+            exp.FileFormatProperty: lambda self, e: f"format={self.sql(e, 'this')}",
```

Comment:
Removing the single quotes is correct here. The Hive engine needs the single quotes but the Trino engine throws an error if you use them.

In addition, I lowercased the property for consistency with every other property